rules/native_binary.bzl

```diff
@@ -17,17 +17,16 @@
 These rules let you wrap a pre-built binary or script in a conventional binary
 and test rule respectively. They fulfill the same goal as sh_binary and sh_test
 do, but they run the wrapped binary directly, instead of through Bash, so they
-don't depend on Bash and work with --shell_exectuable="".
+don't depend on Bash and work with --shell_executable="".
 """
 
-load("//rules/private:copy_file_private.bzl", "copy_bash", "copy_cmd")
-
-def _impl_rule(ctx, is_windows):
+def _impl_rule(ctx):
     out = ctx.actions.declare_file(ctx.attr.out)
-    if is_windows:
-        copy_cmd(ctx, ctx.executable.src, out)
-    else:
-        copy_bash(ctx, ctx.executable.src, out)
+    ctx.actions.symlink(
+        target_file = ctx.file.src,
```

Comment:
From https://buildkite.com/bazel/bazel-skylib/builds/2115#a7024d75-d3b5-4cff-a048-c207efbb6c0b:

```
ERROR: /workdir/tests/native_binary/BUILD:100:12: in native_test rule //tests/native_binary:data_test:
Traceback (most recent call last):
	File "/Users/arostovtsev/bazelbuild/bazel-skylib/rules/native_binary.bzl", line 26, column 31, in _impl_rule
		target_file = ctx.file.src,
Error: No attribute 'src' in file. Make sure there is a label type attribute marked as 'allow_single_file' with this name
```

Note that https://github.com/bazelbuild/bazel-skylib/pull/341 changed `src` to be `allow_files`.

Reply:
Whoops. Missed that when merging. Fixed